litigious-liberators/onboarding/onboarding.py

```diff
@@ -0,0 +1,136 @@
+from kivy.app import App
+from kivy.uix.label import Label
+from kivy.uix.floatlayout import FloatLayout
+from kivy.uix.screenmanager import ScreenManager, Screen
+from kivy.animation import Animation
+from kivy.clock import Clock
+from functools import partial
+
+lorem = ""
+sentences = [
+    "It's the year 12020 CE...",
+    """Civilization has fallen, after the humans caused their own demise as they tried to"""
+    """ interfere with the natural order of things thousands of years ago """,
+    """Experiments with the prehistoric animal DNA, race to the ultimate doomsday"""
+    """ weapon and the total neglect of Global Warming...""",
+    "Today, flora and fauna from all ages flourish, treading the ruins of an ancient civilization.",
+    """Some humans survived against all odds, striving to rebuild what was lost,"""
+    """ but without lost knowledge and wisdom it seems hopeless.""",
+    """You are one of these humans. Will you be the one, cunning and smart enough to tip"""
+    """ the scales or will your ignorant careless attitude ruin everything?""",
+    "Only time will tell.",
+]
+# images = [
+#     "https://placekitten.com/g/1080/1920",
+#     "https://placekitten.com/g/200/300",
+#     "https://placekitten.com/g/300/400",
+# ]
+
+images = ["onboarding/intro_img_crop.jpg"]
+
+
+class MyOnboardWidget(FloatLayout):
+    steps = 0
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.steps = 0
+        # for index, image_url in enumerate(images):
+        #     image_object = AsyncImage(source=image_url, size_hint=(1, 1))
+        #     image_object.opacity = 1 if index == 0 else 0
+        #     setattr(self, f"image_{index}", image_object)
+        #     self.add_widget(image_object)
+
+        for index, sentence in enumerate(sentences):
+            label_object = Label(
+                text=sentence,
+                text_size=(self.width * 4, None),
+                pos_hint={"center_x": 0.50, "center_y": 0.5},
+                color=[1, 1, 1, 1],
+                valign="center",
+                # align="middle",
+                font_size="35sp",
+                font_name="delinquent-black-font/DelinquentExtract-V4we.ttf",
+                # text_size=(400, 600),
+            )
+            # label_object.opacity = 1 if index == 0 else 0
+            label_object.opacity = 0
+            setattr(self, f"description_{index}", label_object)
```

Comment:
A dictionary should be used here to store your labels, or a list, not class attributes.